app/helpers/poem_helper.rb

```diff
@@ -1,8 +1,23 @@
 module PoemHelper
+  # TODO: 他の箇所でも使うようになったら(コメントとか)
+  #       ここじゃない場所に移した方が良さそう
```

Comment:
__[rubocop]__
* Use only ascii symbols in comments. [:ref](https://github.com/bbatsov/ruby-style-guide#english-comments)